pkg/controller/dnszone/dnszone_controller.go

```diff
@@ -2,7 +2,9 @@ package dnszone
 
 import (
 	"context"
+	"errors"
```

Comment:
I would prefer that we use "github.com/pkg/errors" as it includes some more functionality than the stock "error" package.